packages/webui/src/components/ui/TraceListRow.tsx

```diff
@@ -27,8 +29,8 @@ export default function TraceListRow({ trace }: { trace: Trace }) {
         <div className="font-semibold" data-test-id="column-data-method-cell">
           {trace.http?.method.toUpperCase()}
         </div>
-        <div className="font-semibold text-xs text-opacity-70" data-test-id="column-data-code-cell">
-          {trace.http?.statusCode}
+        <div className="font-semibold text-xs text-opacity-70 uppercase" data-test-id="column-data-code-cell">
+          {trace.http?.state === HttpRequestState.Aborted ? 'Aborted' : trace.http?.statusCode}
```

Comment:
🤔 Perhaps in preparation for other incomplete request types, we could extract this logic to a `getResponseResultValue` (or similar) function.

Reply:
Good shout. Fixed https://github.com/FormidableLabs/envy/pull/158/commits/a385a3ce7c627775489e3eddbb3c999e9bbbf2cf - let me know if the naming is bad

Reply:
Your naming is perfect 👍